dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -195,7 +195,7 @@
             <SolidColorBrush x:Key="SolidBackgroundFillColorTertiaryBrush" Color="{StaticResource SolidBackgroundFillColorTertiary}" />
             <SolidColorBrush x:Key="SolidBackgroundFillColorQuarternaryBrush" Color="{StaticResource SolidBackgroundFillColorQuarternary}" />
 
-            <SolidColorBrush x:Key="SystemFillColorAttentionBrush" Color="{StaticResource SystemFillColorAttention}" />
+            <SolidColorBrush x:Key="SystemFillColorAttentionBrush" Color="{ThemeResource SystemAccentColorLight2}" />
```

Comment:
Just curious: why use ThemeResource in Theme dictionary? If using StaticResource here, shouldn't it work the same way because `SystemFillColorAttentionBrush` will be different for each theme?:)

Reply:
Seva is correct, using the ThemeResource markup extension doesn't make sense here.

Reply:
please use StaticResource instead

Reply:
It seems that StaticResource is only evaluated at run-time, so the brush won't update when toggling through different system colour themes. Thus, we have to use ThemeResource here.

Reply:
The Brush is defined in a theme dictionary, so it is only used for one theme. In this case dark. You don't need it to update based on the requested theme. You do however need it to update when the user changes their accent color. So while ThemeResource doesn't make any sense to use inside of a theme dictionary, because you dont need to respond to theme changes from within a theme dictionary, we don't have any resource extension that allows for redefinition execept theme resource.  So yes you can leave this as is.

@YuliKl This is likely the cause of the slider pressed thumb not respecting changed accent colors.

Reply:
This one: https://github.com/microsoft/microsoft-ui-xaml/issues/4568